com.unity.render-pipelines.high-definition/Editor/PostProcessing/LiftGammaGainEditor.cs

```diff
@@ -26,11 +30,11 @@ public override void OnInspectorGUI()
         {
             using (new EditorGUILayout.HorizontalScope())
             {
-                m_TrackballUIDrawer.OnGUI(m_Lift.value, m_Lift.overrideState, EditorGUIUtility.TrTextContent("Lift"), GetLiftValue);
+                m_TrackballUIDrawer.OnGUI(m_Lift.value, m_Lift.overrideState, m_LiftLabel, GetLiftValue);
                 GUILayout.Space(4f);
-                m_TrackballUIDrawer.OnGUI(m_Gamma.value, m_Gamma.overrideState, EditorGUIUtility.TrTextContent("Gamma"), GetLiftValue);
+                m_TrackballUIDrawer.OnGUI(m_Gamma.value, m_Gamma.overrideState, m_GammaLabel, GetLiftValue);
```

Comment:
Why is the last param "GetLiftValue" fro all three of these?

Reply:
Because of bad naming. Don't want to change the function name in this PR. The function is called in the `m_TrackballUIDrawer.OnGUI()` function and it's basically just doing a general vector math
`static Vector3 GetLiftValue(Vector4 x) => new Vector3(x.x + x.w, x.y + x.w, x.z + x.w);`